docs/user/rstudio/ide/guide/tools/terminal.qmd

```diff
@@ -2,93 +2,100 @@
-A terminal with child processes is considered *busy* by RStudio, and will have **(busy)** next to its name, and the close \[x\] changes to a **stop** button.
+A terminal with child processes is considered *busy* by RStudio, and will have:
+
+1. **(busy)** next to the terminal name.
+2. The close **x** changes to a **STOP** button.
+
+![](images/terminal-close.png){fig-alt="The RStudio integrated terminal, highlighting the 'x' button to close."}
 
 ![screenshot showing a busy terminal](images/terminal-sleep.png){fig-alt="screenshot showing a busy terminal"}
 
-This "busy detection" feature is only available on Mac, Linux, and RStudio Workbench / RStudio Server and not on Windows. See [Appendix A - Platform Specifics](https://support.rstudio.com/hc/en-us/articles/115010737148-Using-the-RStudio-Terminal-in-the-RStudio-IDE#appendixa) for more details.
+This "busy detection" feature is only available on Mac, Linux, and Posit Workbench / RStudio Server and not on Windows.
 
 ## Closing Terminal Sessions
 
-To close a terminal session, use the **Close Terminal** command on the Terminal dropdown menu, click the \[x\] on the far-right of the Terminal pane toolbar, or type **exit** from within the shell itself.
+To close a terminal session, use the **Close Terminal** command on the Terminal drop-down menu, click the **x** on the far-right of the Terminal pane toolbar, or type **exit** from within the shell itself.
 
-As discussed earlier, if a terminal is currently busy, the \[x\] changes to a **stop** **button**.
+As discussed earlier, if a terminal is currently busy, the **x** changes to a **STOP** button.
 
-Clicking the **stop button** will trigger the **Interrupt Current Terminal** command, sending an interrupt to the terminal; this may close the currently running program (the program itself decides what to do when receiving an interrupt), but will not close the terminal.
+Clicking the **STOP** button will trigger the **Interrupt Current Terminal** command, sending an interrupt to the terminal; this may close the currently running program (the program itself decides what to do when receiving an interrupt), but will not close the terminal.
 
-Finally, if the terminal feature it not useful to your workflow, you can remove the tab altogether by clicking the \[x\] on the Terminal tab itself. This will close all running terminal sessions, including any custom-named terminal sessions. Subsequent instances of RStudio will not show the Terminal tab: to bring it back, use **Shift+Alt+T** (**Tools-\>Terminal-\>Move Focus to Terminal**).
+Finally, if the terminal feature it not useful to your workflow, you can remove the tab altogether by clicking the **x** on the Terminal tab itself. This will close all running terminal sessions, including any custom-named terminal sessions. Subsequent instances of RStudio will not show the Terminal tab: to bring it back, use **Tools** \> **Terminal** \> **Move Focus to Terminal** (keyboard shortcut: <kbd>Shift+Alt+T</kbd>).
 
```

Comment:
shortcut may need to be updated to shift + alt + m (see above comment)

Reply:
Changed - thanks!